src/mangosd/mangosd.conf.dist.in

```diff
@@ -2812,16 +2812,20 @@ OfflineChat.IP     = 0.0.0.0
 OfflineChat.Password= "p8zqkt"
 
 ###################################################################################################################
-#    Player bots
+#    Bots
```

Comment:
why did you remove the word player, bring it back

Reply:
Because a Bot can be either a PlayerBot or a PartyBot, this way the configuration is easily understandable:

- `Bot`: Configuration affecting both PlayerBot and PartyBot.
- `PlayerBot`: Only affects PlayerBot.
- `PartyBot`: Only affects PartyBot.